text-to-speech/v1.ts

```diff
@@ -70,31 +71,51 @@ class TextToSpeechV1 extends GeneratedTextToSpeechV1 {
    * @param {Object} params The parameters
    * @return {SynthesizeStream}
    */
-  synthesizeUsingWebSocket(params) {
-    params = params || {};
-    params.url = this.baseOptions.url;
-
-    // pass the Authenticator to the SynthesizeStream object
-    params.authenticator = this.getAuthenticator();
-
-    // if the user configured a custom https client, use it in the websocket method
-    // let httpsAgent take precedence, default to null
-    params.agent = this.baseOptions.httpsAgent || this.baseOptions.httpAgent || null;
+  synthesizeUsingWebSocket(params: TextToSpeechV1.SynthesizeWebSocketParams): SynthesizeStream {
+    const streamParams: SynthesizeStream.Options = extend(
+      params,
+      {},
```

Comment:
This should maintain the prior desired behavior of `params || {}` without typescript complaining about how {} is not assignable to WebSocketParams or Options due to messing required keys:
```
$ node
> const extend = require('extend');
> extend(null, {});
{}
> extend({}, null);
{}
> extend({}, 'aa')
{ '0': 'a', '1': 'a' }
> extend('aa', {})
{}
> extend('', {})
{}
> extend({}, '')
{}
```

Reply:
I like this - would you mind making a similar change in the STT code for consistency?

Reply:
Sure, done!